camera/camera_viewer_bq.cc

```diff
@@ -0,0 +1,37 @@
+//%bin(camera_viewer_main)
+#include "infrastructure/balsa_queue/bq_main_macro.hh"
+#include "camera/camera_viewer_bq.hh"
+
+//%deps(balsa_queue)
```

Comment:
When you remove the chevrons from the other file, you can delete this dependency